packages/devextreme/js/__internal/ui/chat/messagegroup.ts

```diff
@@ -175,9 +190,53 @@ class MessageGroup extends Widget<Properties> {
       }
     }
 
+    if (isEdited && alignment === 'start') {
```

Comment:
I suggest extrect alignment === start in separate method
isStartAlignment() { ... }